models/access_token.js

```diff
@@ -1,23 +1,25 @@
 const Sequelize = require("sequelize");
 const sequelize = require("../database/db");
-const User = require("./users");
-
-const Access_Token = sequelize.define("Access_Tokens", {
-  user_id: {
-    type: Sequelize.INTEGER,
-    unique: true,
-    references: {
-      model: User,
-      key: "id",
+const Token = sequelize.define(
+  "accessToken",
+  {
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
```

Comment:
userId should be unique, 1 user should not be able to create multiple token records @akaashinno